web_ui/server.py

```
@@ -3962,6 +4032,32 @@ def get_trackers():
+    example_trackers_raw = example_config.get("TRACKERS", {})
+    example_trackers = cast(dict[str, Any], example_trackers_raw) if isinstance(example_trackers_raw, Mapping) else {}
+
+    cookie_trackers: set[str] = set()
+    try:
+        from src.cookie_auth import find_cookie_file
+
+        for tracker_name in tracker_class_map:
+            cookie_path = Path(find_cookie_file(str(STATE_DIR), tracker_name, user_config))
+            if cookie_path.is_file():
+                cookie_trackers.add(tracker_name.upper())
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)